### ISSUE TITLE
Lines

### ISSUE THREAD

**Issue by bthada** [stephenturner/qqman#23]
Stephen, qqman has been great help in making Manhattan plots of my results. I would like to add vertical lines for things like candidate genes and horizontal lines for a comparison with a QTL analysis. Currently I am able to do it when using a subset of just one chromosome but it gives me "Error: unexpected input..." when trying to use it for the whole genome wide analysis. Any help would be much appreciated. My current code for the single chromosome is:

manhattan(subset(ratioresults, CHR==1), highlight=snpsofinterest)
lines(c(122,145),c(6,6),col="green",lwd=2)
lines(c(32.9,56.62),c(6,6),col="green",lwd=2)
abline(v=50,col="green")

An image of the single chromosome and they types of lines I am talking about:

![image](https://cloud.githubusercontent.com/assets/11278177/6448848/bc8ef0d2-c0ea-11e4-8aa6-ee79d5881eee.png)

Thanks,
Brad

**Comment by stephenturner** [comment]
Works fine for me. Using built-in data:

```coffee
library(qqman)
head(gwasResults)
head(snpsOfInterest)
manhattan(gwasResults, highlight=snpsOfInterest)
lines(c(1500,2500),c(2,2),col="hotpink",lwd=3)
abline(v=2000,col="hotpink", lwd=2)
```

![rplot](https://cloud.githubusercontent.com/assets/460076/6451257/51998902-c0fb-11e4-937d-4bf881c4bfd9.png)

**Comment by bthada** [comment]
Stephen,

That ended up working for me. For some reason, numbers I used when looking
at the single chromosome where taken as Mb, but for the whole genome it was
in bp. The numbers I was using were too small to the point of not showing
up on the graph. Once I switched my code to numbers 1,000,000x those I used
before, it worked just fine when looking at the whole genome. Once I ran
your code, I was able to see that. Thanks for the help.

Brad Thada

Graduate Research Assistant
Purdue University
Agronomy Department
Plant Breeding and Genetics
https://ag.purdue.edu/agry/directory/Pages/bthada.aspx
